herd/dune

```diff
@@ -5,5 +5,5 @@
    herd
    )
    (public_names herd7)
-   (libraries herdtools)
+   (libraries herdtools str)
```

Comment:
I'd rather avoid this build dependency on str , if possible.

Reply:
do you know an alternative for simple string checking?